resources/wikia/modules/spec/imageServing.spec.js

```diff
@@ -21,8 +21,8 @@ describe('Image Serving', function () {
 		expect(typeof cutOptions.yOffset2).toBe('number');
 	});
 
-	using('image sizes', [
```

Comment:
Hello! Yeah, we are currently using QUnit end to end for testing in Mercury, largely because QUnit was the adopted framework by the Ember Core team. Quickly browsing through the documentation for the data provider pattern that you've linked, we have actually solved that problem in a different way in Mercury. We simply use native array iteration to loop through collections defined in the (same file)[https://github.com/Wikia/mercury/blob/dev/test/specs/server/lib/Utils.js#L210] or for extremely large collections, or objects, we use (fixtures defined in JSON files)[https://github.com/Wikia/mercury/tree/dev/test/fixtures) and access those fixtures in our tests by directly requiring the json file (Speaking of which, we can probably move the test cases from my examples into a fixture haha). I find this solution eloquent and simple enough to cover the use case I've gleamed from skimming over the jasmine-data-provider documentation.